test/benchee/conversion/memory_test.exs

```diff
@@ -52,4 +52,43 @@ defmodule Benchee.Conversion.MemoryTest do
       assert best(@list_with_mostly_megabytes, strategy: :none) == unit_for(:byte)
     end
   end
+
+  @kilobyte_unit unit_for(:kilobyte)
+  @megabyte_unit unit_for(:megabyte)
+  @terabyte_unit unit_for(:terabyte)
+
+  describe ".convert" do
```

Comment:
One of the things we really take pride in for Benchee is having a very high level of test coverage. You might have seen that there's a tool that we use for CI that checks the test coverage, and right now it's considered failing because you've added some code that is currently untested. If you click on that link, it will show you the paths that are currently untested. It would be wonderful if you could cover those code paths. I'd be happy to help with that as well if you want - just let me know and I'll tack a commit on here.

Reply:
With the reduction of function surface area in the latest commit all the code paths should be covered.